--- origsrc/gnome-common-3.7.4/macros2/gnome-autogen.sh	2013-08-06 18:07:21.822513600 -0500
+++ src/gnome-common-3.7.4/macros2/gnome-autogen.sh	2013-08-06 18:07:40.566585700 -0500
@@ -356,6 +356,7 @@ case $REQUIRED_AUTOMAKE_VERSION in
     1.11*) automake_progs="automake-1.13 automake-1.12 automake-1.11" ;;
     1.12*) automake_progs="automake-1.13 automake-1.12" ;;
     1.13*) automake_progs="automake-1.13" ;;
+    1.14*) automake_progs="automake-1.14" ;;
 esac
 version_check automake AUTOMAKE "$automake_progs" $REQUIRED_AUTOMAKE_VERSION \
     "http://ftp.gnu.org/pub/gnu/automake/automake-$REQUIRED_AUTOMAKE_VERSION.tar.gz"
