--- origsrc/gnome-common-2.20.0/macros2/gnome-autogen.sh	2007-09-03 16:14:29.000000000 -0500
+++ src/gnome-common-2.20.0/macros2/gnome-autogen.sh	2008-05-25 23:04:13.984375000 -0500
@@ -305,17 +305,18 @@
 #tell Mandrake autoconf wrapper we want autoconf 2.5x, not 2.13
 WANT_AUTOCONF_2_5=1
 export WANT_AUTOCONF_2_5
-version_check autoconf AUTOCONF 'autoconf2.50 autoconf autoconf-2.53' $REQUIRED_AUTOCONF_VERSION \
+export WANT_AUTOCONF=2.5
+version_check autoconf AUTOCONF 'autoconf autoconf2.50 autoconf-2.53' $REQUIRED_AUTOCONF_VERSION \
     "http://ftp.gnu.org/pub/gnu/autoconf/autoconf-$REQUIRED_AUTOCONF_VERSION.tar.gz" || DIE=1
 AUTOHEADER=`echo $AUTOCONF | sed s/autoconf/autoheader/`
 
 case $REQUIRED_AUTOMAKE_VERSION in
     1.4*) automake_progs="automake-1.4" ;;
-    1.5*) automake_progs="automake-1.10 automake-1.9 automake-1.8 automake-1.7 automake-1.6 automake-1.5" ;;
-    1.6*) automake_progs="automake-1.10 automake-1.9 automake-1.8 automake-1.7 automake-1.6" ;;
-    1.7*) automake_progs="automake-1.10 automake-1.9 automake-1.8 automake-1.7" ;;
-    1.8*) automake_progs="automake-1.10 automake-1.9 automake-1.8" ;;
-    1.9*) automake_progs="automake-1.10 automake-1.9" ;;
+    1.5*) automake_progs="automake-1.5 automake-1.10 automake-1.9 automake-1.8 automake-1.7 automake-1.6" ;;
+    1.6*) automake_progs="automake-1.6 automake-1.10 automake-1.9 automake-1.8 automake-1.7" ;;
+    1.7*) automake_progs="automake-1.7 automake-1.10 automake-1.9 automake-1.8" ;;
+    1.8*) automake_progs="automake-1.8 automake-1.10 automake-1.9" ;;
+    1.9*) automake_progs="automake-1.9 automake-1.10" ;;
     1.10*) automake_progs="automake-1.10" ;;
 esac
 version_check automake AUTOMAKE "$automake_progs" $REQUIRED_AUTOMAKE_VERSION \
